### DISCUSSION TITLE
Can two suffixes be grouped together, sharing a description to save space?

### DISCUSSION THREAD

**Discussion by kings2u** [magit/transient#177] [Support] [answered]
The title states my question as succinctly as possible, but here is a visual of how I picture the transient to look:
```
w, s "Move Point Up/Down"
```
Here, `w` would move point up and `s` would move point down. The shortcut keys are placed on the same entry slot in the transient menu and share a description, as opposed to having their own line like usual:
```
w "Move Point Up"
s "Move Point Down"
```
Is something like this possible?

**Comment by tarsius** [comment, marked as answer]
That isn't possible, sorry.

For some value of "possible" of course. You could define your own group class which can be instructed to neither insert a newline character nor align to the next column for certain suffixes. I don't think its worth it.

I might eventually add support for arranging suffixes similar to how hydra does it, but not anytime soon.
